pkg/snet/network.go

```diff
@@ -165,8 +168,16 @@ func New(conf Config, eb *appevent.Broadcaster, masterLogger *logging.MasterLogg
 			SK:              conf.SecKey,
 			AddressResolver: conf.ARClient,
 		}
-
-		clients.Direct[tptypes.SUDPH] = directtp.NewClient(sudphConf, masterLogger)
+		// SUDPH wont be initilised only if the visor is under any symmetric NAT
+		// along with the other NAT types all other errors will also initilise the SUDPH
+		// as without a proper response from a STUN server we cannot determine if the
+		// visor is un der any symmetric NAT
+		switch conf.NATType {
+		case stun.NATSymmetric, stun.NATSymmetricUDPFirewall:
+			break
```

Comment:
I would add an Info level log message here that sudph transport is not available because of NAT

Reply:
Good catch. Will do.